index.html

```diff
@@ -99,306 +152,98 @@ <h1>
+              <div class="form-container">
+                <form action="https://formspree.io/f/mleanrpl" method="post">
+                  <ul class="list-form">
+                    <li class="list4">
+                      <input
+                        type="text"
+                        id="name"
+                        maxlength="30"
+                        name="user_name"
+                        placeholder="Tyler Durden"
+                        required
+                      />
+                    </li>
+                    <li class="list4">
+                      <input
+                        type="email"
+                        id="mail"
+                        size="30"
+                        name="user_email"
+                        placeholder="...@email.com"
+                        required
+                      />
+                    </li>
+                  </ul>
+                  <textarea
+                    id="msg"
+                    name="user_message"
+                    placeholder="Write your message here"
+                    maxlength="500"
+                    required
+                  ></textarea>
+                  <button type="submit" class="btn btn-transition" id="0">
+                    Get it Touch
+                  </button>
+                </form>
+              </div>
             </div>
-            <div class="form-container">
-              <form action="https://formspree.io/f/mleanrpl" method="post">
-                <ul class="list-form">
-                  <li class="list4">
-                    <input
-                      type="text"
-                      id="name"
-                      maxlength="30"
-                      name="user_name"
-                      placeholder="Tyler Durden"
-                      required
-                    />
-                  </li>
-                  <li class="list4">
-                    <input
-                      type="email"
-                      id="mail"
-                      size="30"
-                      name="user_email"
-                      placeholder="...@email.com"
-                      required
-                    />
-                  </li>
-                </ul>
-                <textarea
-                  id="msg"
-                  name="user_message"
-                  placeholder="Write your message here"
-                  maxlength="500"
-                  required
-                ></textarea>
-                <button type="submit" class="btn btn-transition">
-                  Get it Touch
-                </button>
-              </form>
+            <div class="barxx">
+              <div class="bar12"></div>
+             </div>
             </div>
-          </div>
-          <div class="barxx">
-            <div class="bar12"></div>
-          </div>
-        </div>
-      </section>
-    </main>
+        </section>
+      </main>
+    </div>
+    <section class="popup"></section>
```

Comment:
- [ ] **As the previous reviewer pointed out, the project images are not loaded in the project pop-up. Please fix this**

The images are missing due to a bad path. Please fix the path to images. 

![image](https://user-images.githubusercontent.com/59168430/123308184-fc277680-d540-11eb-8a0d-71d3deb64d22.png)